sig-cluster-lifecycle/annual-report-2022.md

```diff
@@ -4,64 +4,93 @@
+   our other subprojects are fairly new in Kubernetes years.
 
 3. KEP work in 2022 (v1.24, v1.25, v1.26):
 
+As noted above, only KEP work for kubeadm was done in 2021.
+Some KEPs were retroactively updated to GA, since the work there was done without KEP updates.
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - ["kubernetes-sigs/cluster-api-addon-provider-helm"](https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/blob/main/OWNERS)
+   - [etcdadm](https://github.com/kubernetes-sigs/etcdadm)
+   - [cluster-addons](https://github.com/kubernetes-sigs/cluster-addons)
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - We started collecting metrics from all of our subprojects using an
+     [yearly survey](https://forms.gle/xZn8DXww4XxPsXvCA). It contains a number of questions
+     related to project health, OWNERS files, contributor onboarding, etc. What we saw for this annual
+     report is that our OWNERS files and SIG README.md are mostly up-to-date and that subprojects are doing
+     what they can to onboard new contributors. We are considering preparing some actions to better educate
+     our subprojects about onboarding contributors and graduating more OWNERS. We did not do a survey
+     for 2022.
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - As per the 2021 survey, we are seeing a good number of projects to keep their contributing
+     documentation up-to-date. We are drafting action items for the leads to improve the understanding
+     around [CONTRIBUTING.md] management. We have a good number of projects that have participated in LFX and GSoC.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - Our subprojects have implemented their own contributing process and they have their own criteria
+     for contributors to climb the ladders. We have not seen complains from new or existing
+     contributors related to contributor guides. The SIG leads are open to help subproject leads with tips
+     on this topic.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, as confirmed by our survey.
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   In our survey results we found a few common areas where more help is needed across subprojects:
```

Comment:
how do we know these are still relevant if we didn't do a survey in over a year?

Reply:
That's the best approximation we have, but happy to change the list if you think some area deserve more visibility